src/lib/holocene/accordion.svelte

```diff
@@ -75,12 +75,14 @@
         <slot name="action" />
       </div>
       {#if !readOnly}
-        <Icon
-          name={open ? 'chevron-up' : 'chevron-down'}
-          class="rounded-full from-blue-100 to-purple-100 hover:bg-gradient-to-br dark:from-blue-800 dark:to-purple-800 {disabled
-            ? 'text-disabled'
-            : 'text-primary'}"
-        />
+        <div
```

Comment:
Because icons changed to 16×16px, there were some rendering issues with this one. I wrapped the chevron icon inside a div to make it appear the same as before.